ccm_web/client/src/components/UserEnrollmentForm.tsx

```diff
@@ -0,0 +1,387 @@
+                          Finish the form to enroll them in the section.
+                        </Typography>
+                      )
+                }
+              </Paper>
+            )
+          }
+          <div className={classes.spacing}>
+            {
+              showIncompleteAlerts && role === undefined &&
+                <InlineErrorAlert>You must select a Canvas role from the dropdown.</InlineErrorAlert>
+            }
+            <RoleSelect
+              roles={props.rolesUserCanEnroll}
+              selectedRole={role}
+              onRoleChange={setRole}
+              disabled={isEnrollmentLoading}
+            />
+          </div>
+          <Typography className={classes.spacing}>
+            Select the section you want to enroll the user in.
+          </Typography>
+          {
+            showIncompleteAlerts && selectedSection === undefined &&
+              <InlineErrorAlert>You must select one section from the list below.</InlineErrorAlert>
+          }
+          <SectionSelectorWidget
+            height={300}
+            search={[]}
+            multiSelect={false}
+            sections={props.sections}
+            selectedSections={selectedSection !== undefined ? [selectedSection] : []}
+            selectionUpdated={(sections) => {
+              if (sections.length === 0) {
+                setSelectedSection(undefined)
+              } else {
+                setSelectedSection(sections[0])
+              }
+            }}
+            canUnmerge={false}
+          />
+          <Grid container className={classes.buttonGroup} justifyContent='space-between'>
+            <Button
+              variant='outlined'
+              aria-label='Back to input method select'
+              onClick={props.resetFeature}
+            >
+              Back
+            </Button>
+            <Button
+              color='primary'
+              variant='contained'
+              disabled={isLoading}
+              aria-label='Submit single user enrollment'
+              onClick={handleSubmitClick}
+            >
+              Submit
+            </Button>
+          </Grid>
+        </Grid>
+        <Backdrop className={classes.backdrop} open={isEnrollmentLoading}>
+          <Grid container>
+            <Grid item xs={12}>
+              <CircularProgress color='inherit' />
+            </Grid>
+            <Grid item xs={12}>Sending user and enrollment data to Canvas</Grid>
+          </Grid>
+        </Backdrop>
+      </Grid>
+    )
+  }
+
+  const renderSuccess = (userExists: boolean): JSX.Element => {
+    const settingsLink = (
+      <Link href={props.settingsURL} target='_parent'>Canvas Settings page</Link>
+    )
+    const messageText = (
+      userExists
+        ? 'The existing user was'
+        : 'The new user was invited to create a friend account, added to Canvas, and'
+    ) + ' enrolled in the selected section!'
```

Comment:
Update as per @melindakraft `The new user was invited to choose a login method, added to Canvas, and enrolled in the selected section.`

Reply:
Done. See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/270/commits/be08a25d4f0268b266238d997c5c5324ccec07ac